ml-proto/host/encode.ml

```diff
@@ -0,0 +1,417 @@
+(* Version *)
+
+let version = 11
```

Comment:
Elsewhere, the project documents the version as hex, so this would be more clear as `0xb`.